### PR TITLE
Add .defined_schema method

### PR DESCRIPTION
**Pull request by glaucocustodio**

### CODE REVIEW

File: lib/surrealist/class_methods.rb [hunk 1]
```diff
@@ -55,6 +55,26 @@ def json_schema(&_block)
...
+    #
+    #     def name
+    #       'Parent'
+    #     end
+    #   end
+    #
+    #   Person.defined_schema
+    #   # => { name: String }
+    def defined_schema
+      instance_variable_get('@__surrealist_schema')
```
Comment (glaucocustodio) [inline lib/surrealist/class_methods.rb]: Should we raise `UnknownSchemaError` or any other error if any schema has been defined?

Comment (nesaulov) [reply lib/surrealist/class_methods.rb]: Yes, I think we should

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# Next (unreleased)
+
+## Added
+* `.defined_schema` to return the schema that has been defined with `json_schema` ([@glaucocustodio][]) [#93](https://github.com/nesaulov/surrealist/pull/98)
```
Comment (nesaulov) [inline CHANGELOG.md]: You should add a link to yourself at the end of the file, see line 95

File: spec/support/schema_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+WithoutSchema = Class.new do
+  include Surrealist
+end
+
+RSpec.describe Surrealist do
+  describe '.defined_schema' do
+    context 'json schema defined' do
+      let(:instance) { WithSchema }
+      it do
```
Comment (nesaulov) [inline spec/support/schema_spec.rb]: A space between `let` and `it` is missing.
Also I would suggest to add a specification to `it`, like `it 'returns the defined json_schema' do`

File: spec/support/schema_spec.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        expect(instance.defined_schema).to eq(name: String, age: Integer)
+      end
+    end
+
+    context 'json schema not defined' do
+      let(:instance) { WithoutSchema }
+      it do
```
Comment (nesaulov) [inline spec/support/schema_spec.rb]: same here

**Comment by glaucocustodio** [comment]
```ruby
class CatSerializer < Surrealist::Serializer
  json_schema { { age: Integer, age_group: String } }

  def age_group
    age <= 5 ? 'kitten' : 'cat'
  end
end
```

```ruby
class Cat
  include Surrealist
  attr_reader :age

  surrealize_with CatSerializer

  def initialize(age)
    @age = age
  end
end
```

In this case of custom serializer, should `Cat` respond to `defined_schema` as well?

**Comment by glaucocustodio** [comment]
Take a look now please @nesaulov

**Comment by glaucocustodio** [comment]
Done @nesaulov

**Comment by nesaulov** [comment]
🤔 there is something wrong with rspec or rom there

**Comment by nesaulov** [comment]
The problem was with dry-types and rom-rb compatibility, see rom-rb/rom#494
I have locked dry-types version for now in #101, should be fine now.

**Comment by nesaulov** [comment]
Thank you @glaucocustodio! 🎉 
Sorry that this took so long

**Pull request merged**